test-runner/src/reporters/base.ts

```diff
@@ -115,11 +111,13 @@ export class BaseReporter implements Reporter  {
       this._printFailures(this.flaky);
     }
 
-    if (this.timedOut.length) {
-      console.log(colors.red(`  ${this.timedOut.length} timed out`));
+    const timedOut = this.unexpected.filter(t => t._hasResultWithStatus('timedOut'));
```

Comment:
So, we can report a single test twice - as failed and timed out. By design?

Reply:
We can, not sure what to about it. Maybe I should report all as failures and annotate as a timeout.